components/combobox/index.css

```diff
@@ -0,0 +1,609 @@
+  font-family: var(--mod-combobox-font-family, var(--spectrum-combobox-font-family));
+  font-weight: var(--mod-combobox-font-weight, var(--spectrum-combobox-font-weight));
+  line-height: var(--mod-combobox-line-height, var(--spectrum-combobox-line-height));
+  font-style: var(--mod-combobox-font-style, var(--spectrum-combobox-font-style));
+  color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-default, var(--spectrum-combobox-font-color-default)));
+
+  /* fill space */
+  flex: 1;
+
+  &::placeholder {
+    color: var(--highcontrast-combobox-font-color-placeholder, var(--mod-combobox-font-color-placeholder, var(--spectrum-combobox-font-color-placeholder)));
+  }
+
+  &:hover,
+  &:active {
+    background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-hover, var(--spectrum-combobox-background-color-hover)));
+    border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-hover, var(--spectrum-combobox-border-color-hover)));
+    color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-hover, var(--spectrum-combobox-font-color-hover)));
+
+    &::placeholder {
+      color: var(--highcontrast-combobox-font-color-placeholder, var(--mod-combobox-font-color-hover, var(--spectrum-combobox-font-color-hover)));
+    }
+  }
+
+  &:focus,
+  &.is-focused {
+    background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-focus, var(--spectrum-combobox-background-color-focus)));
+    border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-focus, var(--spectrum-combobox-border-color-focus)));
+    color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-focus, var(--spectrum-combobox-font-color-focus)));
+
+    &::placeholder {
+      color: var(--highcontrast-combobox-font-color-placeholder, var(--mod-combobox-font-color-focus, var(--spectrum-combobox-font-color-focus)));
+    }
+  }
+
+  &:focus:hover,
+  &.is-focused:hover {
+    background-color: var(--highcontrast-combobox-background-color, var(--mod-combobox-background-color-focus-hover, var(--spectrum-combobox-background-color-focus-hover)));
+    border-color: var(--highcontrast-combobox-focus-indicator-color, var(--mod-combobox-border-color-focus-hover, var(--spectrum-combobox-border-color-focus-hover)));
+    color: var(--highcontrast-combobox-font-color, var(--mod-combobox-font-color-focus-hover, var(--spectrum-combobox-font-color-focus-hover)));
+  }
+}
+
+.spectrum-Combobox--quiet {
+  border-radius: var(--mod-combobox-quiet-fieldbutton-border-radius, var(--spectrum-combobox-quiet-fieldbutton-border-radius));
+
+  .spectrum-Combobox-textfield {
+    &.is-keyboardFocused::after {
+      pointer-events: none;
+      content: '';
+      position: absolute;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      top: 0;
```

Comment:
All of this is a duplicate of the styles either from Textfield or that it already has before being quiet so you don't need these